cloudformation-custom-resource/src/main/scala/feral/cloudformation/package.scala

```diff
@@ -0,0 +1,111 @@
+package feral
+
+import cats.implicits._
+import shapeless.tag.@@
+import io.circe._
+import io.circe.syntax._
+import io.circe.generic.semiauto._
+
+package object cloudformation {
```

Comment:
Can some of this stuff be moved to the `events` module maybe? Perhaps a bit over-modulization, but I think its valuable to offer our case class encodings of the basic AWS event types so people can use them with their own custom `IOLambda`s without buying into our implementations.

Reply:
I went back and forth on using the `events` module. I see where you're coming from by allowing others to use the models, but it also means as a user of just the CloudFormation Custom Resource class, I have a lot of models in scope that actually aren't relevant to that task. Tradeoffs either way, I think.

Reply:
I agree, certainly tradeoffs either way, definitely open to ideas for how to best tackle this. FWIW the `events` module is akin to [aws-lambda-java-events](https://github.com/aws/aws-lambda-java-libs/tree/master/aws-lambda-java-events) and [aws-lambda/trigger](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/aws-lambda/trigger). Maybe we could scope this a bit better with additional packages?

Also, to explain my thinking a bit, I tend to determine modules by their dependencies. Taking the JS implementations as an example:
- `core`: CE3
- `lambda`: `core` + `circe`
- `events`: `circe-generic` (i.e. shapeless :)
- `lambdaApiGatewayProxyHttp4s`: `lambda` + `events` + `http4s-core`
- `lambdaCloudFormationCustomResource`: `lambda` + `events` + `http4s-ember-client`

So, the `events` module essentially lets users write lambdas without having http4s (or even CE3!) in scope if that's not relevant to what they want to do either.

I think it ultimately comes down to priorities, if we pretty much assume people will be doing things our way then I totally agree with you, no need for an `events` module and we can keep things scoped.

Reply:
My preference is to leave these things in the CloudFormation module, so I left this alone for now. I think if people aren't doing things our way they're more likely to just use aws-lambda-java-events or whatever, and if they are using our core then having a more focused set of model classes makes sense. But either way is fine, so if there's consensus to move them into events, I can do that too.